public/js/main.js

```diff
@@ -0,0 +1,27 @@
+let loginBtn = document.querySelector(".login-button");
+let createBtn = document.querySelector(".create-button");
+let playBtn = document.querySelector(".nav-link-play");
+let instructions = document.querySelector(".instructions");
+let game = document.querySelector(".game");
+let instructionsStartGame = document.querySelector(".start-button");
+let statsBtn = document.querySelector(".nav-link-stat");
+
+let showInstructions = () => {
+  instructions.classList.remove("hidden");
+  let form = document.querySelector(".login-form");
+  form.classList.add("hide-form");
```

Comment:
I think you want to hide the form before you show the instructions, and why not just add the hidden class that we already have to this?

Reply:
The code I wrote hides the form when you fill it out. It then shows the instructions. Then when you click the play button on the instructions it hidesvthat too.

Reply:
The code I wrote hides the form when you fill it out. It then shows the instructions. Then when you click the play button on the instructions it hidesvthat too.